concordium-node/src/configuration.rs

```diff
@@ -76,6 +76,10 @@ pub const SOFT_BAN_DURATION_SECS: u64 = 300;
 pub const MAX_PEER_NETWORKS: usize = 20;
 /// Database subdirectory name
 pub const DATABASE_SUB_DIRECTORY_NAME: &str = "database-v4";
+/// Default out-of-band catch-up file name (excluding the extension)
+pub const CATCHUP_FILE_BASENAME: &str = "blocks_to_import";
+/// Default out-of-band catch-up file extension
+pub const CATCHUP_FILE_EXT: &str = ".mdb";
```

Comment:
I don't think we should use the extension "mdb", because it suggests it's an LMDB database or a Microsoft Access database, and it's neither of those things. Perhaps a more neutral "db".